code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,234 @@
 			new /obj/item/clothing/mask/luchador(src)
 		if(100)
 			new /obj/item/clothing/head/bearpelt(src)
+		if(101 to 105)
+			var/ctype = pick(subtypesof(/datum/supply_pack))
+			var/datum/supply_pack/pack = new ctype()
+			for(var/item in pack.contains)
+				new item(src)
+			qdel(pack)
+		if(106)
+			new /obj/item/reagent_containers/glass/bottle/mannitol(src)
+			new /obj/item/storage/pill_bottle/neurine(src)
+			new /obj/item/organ/brain(src)
+		if(107)
+			new /obj/item/reagent_containers/glass/bottle/brainrot(src)
+			new /obj/item/reagent_containers/glass/bottle/gbs(src)
+			new /obj/item/reagent_containers/glass/bottle/tuberculosis(src)
+			new /obj/item/reagent_containers/glass/bottle/tuberculosiscure(src)
+			new /obj/item/reagent_containers/glass/bottle/wizarditis(src)
+			new /obj/item/clothing/head/bio_hood/virology(src)
+			new /obj/item/clothing/suit/bio_suit/virology(src)
+			new /obj/item/tank/internals/oxygen/yellow(src)
+			new /obj/item/clothing/gloves/color/latex/nitrile(src)
+		if(108)
+			new /obj/item/circuitboard/machine/protolathe(src)
+		if(109)
+			var/list/circuits = subtypesof(/obj/item/circuitboard)
+			var/chosen_circuit = pick(circuits)
+			new chosen_circuit(src)
+		if(110)
+			new /obj/item/clothing/head/bronze(src)
+			new /obj/item/clothing/shoes/bronze(src)
+			new /obj/item/clothing/suit/bronze(src)
+			new /obj/item/stack/tile/bronze/thirty(src)
+		if(111)
+			new /obj/item/clothing/accessory/medal/bronze_heart(src)
+			new /obj/item/clothing/accessory/medal/conduct(src)
+			new /obj/item/clothing/accessory/medal/plasma(src)
+			new /obj/item/clothing/accessory/medal/plasma/nobel_science(src)
+			new /obj/item/clothing/accessory/medal/ribbon/cargo(src)
+			new /obj/item/clothing/accessory/medal/silver/excellence(src)
+			new /obj/item/clothing/accessory/medal/silver/security(src)
+			new /obj/item/clothing/accessory/medal/silver/valor(src)
+		if(112)
+			new /obj/item/clothing/accessory/medal/gold/heroism(src)
+			new /obj/item/clothing/accessory/medal/gold/captain(src)
+			new /obj/item/reagent_containers/food/drinks/trophy/gold_cup(src)
+			new /obj/item/reagent_containers/food/snacks/grown/apple/gold(src)
+		if(113)//Lite, mid and heavy are Snowndin lootdrops. Putting a spawner into the chest result things spawn on top of it instead of inside.
+			switch(pickweight(list("heavystuff1" = 1, "heavystuff2" = 1, "heavystuff3" = 1, "heavystuff4" = 1, "heavystuff5" = 1, "heavystuff6" = 1, "heavystuff7" = 1, "heavystuff8" = 1)))
+				if("heavystuff1")
+					new /obj/item/twohanded/singularityhammer(src)
+				if("heavystuff2")
+					new /obj/item/twohanded/mjollnir(src)
```

Comment:
Wizard item.

Reply:
wiznerd items ain't the problem. The problem is some are super OP like the mjollnir. A traitor with it is as deadly as a nukie imo.